docs/CHANGELOG.asciidoc

```diff
@@ -26,10 +26,370 @@ Review important information about the {kib} 8.x releases.
+* Adds comparision to service maps popover {kibana-pull}120839[#120839]
+* Link originating service in traces list table {kibana-pull}120768[#120768]
+* Prefer `service.name` for logs correlation {kibana-pull}120694[#120694]
+* Query numerator & denominator simultaneously for log threshold alerts {kibana-pull}107566[#107566]
+
+Operations::
+Improves the file logging capabilities so that missing directories in the configured file path are now created before {kib} attempts to write to the file {kibana-pull}117666[#117666]
+
+Platform::
+* Add a new `elasticsearch.compression` configuration property to enable compression for communications between {kib} and {es} {kibana-pull}124009[#124009]
+* Adds support of comments {kibana-pull}122457[#122457]
+* Adds support for PNG and PDF reports on Darwin Arm64 architecture {kibana-pull}122057[#122057]
+* Short URL client is now accessible on the frontend through plugin contract. *Dashboard* and *Discover* shared short URLs now contain a three word, human-readable slug {kibana-pull}121886[#121886]
+* Adds the ability to add URL drilldowns to *Dashboard* panels {kibana-pull}121801[#121801]
+* Adds a new structure to the report details flyout to help you find information faster {kibana-pull}120617[#120617]
+* Adds HTML tag and impact level to axe-core CI violation reporter {kibana-pull}119903[#119903]
+* Exposes {es} accuracy warnings to the user {kibana-pull}116632[#116632]
+
+Querying & Filtering::
+Improves the version field type {kibana-pull}123739[#123739]
+
+Security::
+Removes the audit space {kibana-pull}124145[#124145]
+
+[float]
+[[fixes-v8.1.0]]
+=== Bug Fixes
+Alerting::
+* Fixes the pagination results for fetching existing alerts {kibana-pull}122474[#122474]
+* Running disabled rules are now skipped {kibana-pull}119239[#119239]
+
+Canvas::
+* Fixes an issue where the image repeat element was not updating {kibana-pull}118701[#118701]
+* Fixes an issue where *Canvas* validated values before saving variables {kibana-pull}118694[#118694]
+
+Dashboard::
+Adds the listing page callout when new dashboards are in progress {kibana-pull}117237[#117237]
+
+Discover::
+* Adds the ability to close the expanded document sidebar when you change data views {kibana-pull}119736[#119736]
+* Fixes search on page load tests {kibana-pull}119087[#119087]
+
+Elastic Security::
+For the Elastic Security 8.1.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Enterprise Search::
+For the Elastic Enterprise Search 8.1.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+
+Fleet::
+* Readded missing packages to keep up to date list {kibana-pull}125787[#125787]
+* Trimmed whitespace from package policy names {kibana-pull}125400[#125400]
+
+Lens & Visualizations::
+* Extracted and injected references for by value in *Visualize Library* {kibana-pull}126499[#126499]
```

Comment:
```suggestion
*  Fixes some dashboard visualizations that could show "Could not located index pattern" errors when copied from one space to another {kibana-pull}126499[#126499]
```

I think this description would be better (I added it to the PR description as well) but I will defer to @flash1293